api/src/pcapi/scheduled_tasks/commands.py

```diff
@@ -102,9 +102,9 @@ def synchronize_cgr_stocks() -> None:
 @blueprint.cli.command("synchronize_ems_stocks")
 @log_cron_with_transaction
 @cron_require_feature(FeatureToggle.ENABLE_EMS_INTEGRATION)
-def synchronize_ems_stocks() -> None:
-    """Launch EMS synchronization"""
-    synchronize_ems_venue_providers()
+def synchronize_ems_stocks_on_schedule() -> None:
```

Comment:
```suggestion
def synchronize_ems_stocks() -> None:
```
(pour être cohérent avec les autres commandes)